workers/tests/unittests/test_topic.py

```diff
@@ -28,6 +28,7 @@ def test_topic_definition():
         indicators=["mapping-saturation"],
         project="core",
         endpoint="endpoint",
+        aggregation_type="aggregation_type",
```

Comment:
After using Literal in the topic model, this will fail.

Reply:
Done